src/main/java/org/jenkinsci/plugins/envinject/service/EnvInjectVariableGetter.java

```diff
@@ -111,6 +112,20 @@
         Map<String, String> triggerVariable = new BuildCauseRetriever().getTriggeredCause(build);
         result.putAll(triggerVariable);
 
+        //Get Cause UserName
+        String userName = new BuildCauseRetriever().getCauseUserName(build);
```

Comment:
I would refactor the code to call <code>new BuildCauseRetriever()</code> only once in order to simplify the life of the garbage collector